src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtImpPrebid.java

```diff
@@ -29,4 +29,16 @@
      */
     @JsonProperty("storedbidresponse")
     List<ExtStoredBidResponse> storedBidResponse;
+
+    /**
+     * Defines the contract for bidrequest.imp[i].ext.prebid.is_rewarded_inventory
+     */
+    @JsonProperty("is_rewarded_inventory")
```

Comment:
This annotation is redundant, since if it is missing Jason will resolve camel case naming (typical Java variable name) into snake case - which is going to be exactly `is_rewarded_inventory`. So you can skip the annotation when variable name is resolved as expected `.json` property name.